sheets/2/exercise-6.tex

```diff
@@ -0,0 +1,74 @@
+\excercise{Wenn ich mich doch nur nach links drehen könnte}
+
+\begin{enumerate}[label=\alph*)]
+    \item Instanziiere den Task und den Verifier. Dabei soll wie in der \textit{Aufgabe 5b)} der SubTask jeweils angegeben werden.
+    \item Lasse Neo im Uhrzeigersinn an der Wand entlang laufen, bis er wieder an seinem Startpunkt angekommen ist.
+    Implementiere das Verhalten in der Methode \textit{subTaskB()} in der Klasse \textit{Sheet3Task6}.
```

Comment:
Methoden wurden an dieser stelle noch nicht eingeführt (auch wenn es in dieser Aufgabe noch passiert) und sollten deshalb noch nicht benutzt werden.